javascript/package.json

```diff
@@ -76,5 +76,9 @@
   "directories": {
     "test": "test"
   },
-  "keywords": []
+  "keywords": [],
+  "dependencies": {
```

Comment:
Everything else seems to be declared as devDependencies. Does fortawesome package are fine as regular dependencies?

Reply:
Sorry - this was a quick experiment - going to make the depedency change in react-components and do a patch release there.